contracts/package.json

```diff
@@ -33,7 +33,7 @@
     "ethereum-waffle": "^3.2.0",
     "ethers": "^5.0.17",
     "fs-extra": "^9.0.1",
-    "hardhat": "^2.6.0",
+    "hardhat": "^2.6.8",
```

Comment:
The warning message is missing a space before the address

<img width="442" alt="Screen Shot 2021-12-08 at 4 14 54 PM" src="https://user-images.githubusercontent.com/1479802/145287450-da70a73b-3161-4713-bb07-e989fc9afeae.png">

Reply:
Fixed in 4c3370c